Main.py

```diff
@@ -0,0 +1,42 @@
+from functions import *
+
+# keyboard
+win.listen()
+win.onkeypress(rocket_left_up, "w")
+win.onkeypress(rocket_left_down, "s")
+win.onkeypress(rocket_right_up, "Up")
+win.onkeypress(rocket_right_down, "Down")
+
+score_a = 0
+score_b = 0
+
+while True:
+    win.update()
+    # ball movements
+    ball.setx(ball.xcor() + ball.dx)
+    ball.sety(ball.ycor() + ball.dy)
+
+    if ball.ycor() > 290:
```

Comment:
All the integer constants in this game are unclear. The constants should have name, like `RIGHT_BOUNDARY = 290`, `LEFT_BOUNDARY = -290` etc.

Another important thing is that all this constants a tight to each other. In other words, you can't change a single constant without changing the other constants if you want your code to keep working without bugs. Therefore, I'd suggest having a class, `Field` or `GameField` (if you want to be more explicit). I imagine it's signature like:

```python
class Field:
    def __init__(self, horizontal_size, vertical_size):
        self.horizontal_size = horizontal_size
        self.vertical_size = vertical_size

    @property
    def center(self):
        return self.horizontal_size / 2, self.vertical_size / 2

    @property
    def top_left(self):
        pass

    @property
    def top_right(self):
         pass

    # Add other important relative coordinates on the field
```

Having such a class will allow you to change field size in a single place without breaking other code.